manifests/foreman.pp

```diff
@@ -55,12 +57,11 @@
     }
 
     file { $ssl_ca_cert:
-      ensure  => file,
-      source  => $server_ca,
-      owner   => 'root',
-      group   => $group,
-      mode    => '0440',
-      require => File[$server_ca],
+      ensure => file,
+      source => pick($server_ca, $certs::ca::server_ca_path),
```

Comment:
@ekohl Thoughts on this approach? With the `undef` optional parameter in the class?

I could just drop the class parameter all together - I don't think it really gets any value or will be used at this point. The `server_ca_path` is tied to the `certs::ca` class as that is what generates / defines it.